src/tests/common/GOTestCollection.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2023-2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+
+#include "GOTestCollection.h"
+#include "GOTestException.h"
+#include "GOTestResultCollection.h"
+#include <iostream>
+#include <iterator>
+#include <vector>
+
+GOTestCollection::GOTestCollection() {}
+
+// GOTestCollection::~GOTestCollection() {
```

Comment:
It's usually bad practice to leave code thats commented out. Consider removing this if it's not going to be used in the very close future.

Reply:
Of course, forgot it.